### DISCUSSION TITLE
I can't unstar repo

### DISCUSSION THREAD

**Discussion by zielvna** [community/community#24625] [Apps, API and Webhooks]
<p>Hi,</p>
<p>I deleted my repository and now I can’t unstar it. I tried unstaring with api but it doesn’t work. How can I do that?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>just to be sure, restore it and see,</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/47092464/181085721-210ba3f4-0fe3-4b92-8745-a1703b16c48d.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository" target="_blank" rel="noopener">Restoring a deleted repository - GitHub Docs</a></h3>

  <p>You can restore some deleted repositories to recover their contents.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>unstar if starred then you can delete it afterwards</p>
